www/js/pages/profile/profile.controller.js

```diff
@@ -67,10 +67,12 @@
 
         vm.init = () => {
             vm.isVirtualAccount = !!appStateService.virtuality;
+            vm.account = accountService.getDefault();
```

Comment:
it doesn't need to add account info to `vm` object since it just is used here in order to check that account is MF or not. so it's better to declare as `const` in if clause.